skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/model/Span.java

```diff
@@ -166,12 +170,12 @@ public void setBusinessKey(String businessKey) {
 
     public RequestSpan.Builder buildRequestSpan(RequestSpan.Builder builder) {
         return builder.setTraceId(this.traceId).setParentLevel(this.parentLevel).setLevelId(this.levelId)
-                .setStartDate(this.startDate).setRouteKey(routeKey).putAllTags(tags);
+                .setStartDate(this.startTime).setRouteKey(routeKey).putAllTags(tags);
```

Comment:
Still have two method names, `StartDate` and `StartTime`, please make sure they are uniform.